code/game/objects/items/weapons/manuals.dm

```diff
@@ -283,6 +283,135 @@
 				Coming soon...
 			"}
 
+/obj/item/weapon/book/manual/experimentor
+	name = "Mentoring your Experiments"
+	icon_state = "rdbook"
+	author = "Dr. H.P. Kritz"
+	dat = {"<html>
+				<head>
+				<style>
+				h1 {font-size: 18px; margin: 15px 0px 5px;}
+				h2 {font-size: 15px; margin: 15px 0px 5px;}
+				li {margin: 2px 0px 2px 15px;}
+				ul {list-style: none; margin: 5px; padding: 0px;}
+				ol {margin: 5px; padding: 0px 15px;}
+				</style>
+				</head>
+				<body>
+				<h1>THE E.X.P.E.R.I-MENTOR</h1>
+				The Enhanced Xenobiological Period Extraction (and) Restoration Instructor is a machine designed to discover the secrets behind every item in existence.
+				With advanced technology, it can process 99.95% of items, and discover their uses and secrets.
+				The E.X.P.E.R.I-MENTOR is a Research machine that takes items, and through a process of elimination, allows you to research technology from them.
+				Due to the volatile nature of the E.X.P.E.R.I-MENTOR, there is a probably result of the machine backfiring and it's subsystems causing havoc.
+				Upgrading the parts of the E.X.P.E.R.I-MENTOR will result in less failiures.
+				The E.X.P.E.R.I-MENTOR uses randomly generated patterns to identify the technological uses of items, meaning that at the start of every shift, it will have a whole new set of uses.

```

Comment:
"Please note that the E.X.P.E.R.I-MENTOR uses a state-of-the-art random generator, which has a larger entropy than the observable universe, therefore it can generate wildly different results each day, therefore it is highly suggested to re-scan objects of interests frequently (e.g. each shift)."